core/pom.xml

```diff
@@ -231,6 +262,19 @@
                 <version>${commons-io.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.squareup.retrofit2</groupId>
+                <artifactId>retrofit</artifactId>
+                <version>${retrofit.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.squareup.retrofit2</groupId>
+                <artifactId>converter-gson</artifactId>
+                <version>${retrofit.version}</version>
+            </dependency>
```

Comment:
@hamzaed Can you explain why we use these new dependencies?

Reply:
I used Retrofit to create the REST client API for NLP.js. This library is de facto API client library for Android and Java. As you can see from the code, You only have to provide a POJO model and an interface with annotated methods that specify your API calls.  converter-gson is in charge of serializing and deserializing json objects.